src/asm/main.c

```diff
@@ -220,6 +220,7 @@ int main(int argc, char *argv[])
 			haltnop = false;
 			break;
 
+		case 'I':
```

Comment:
The fact that this PR leaves `-i` undocumented despite being the original option, and not deprecated, makes me wonder if we should really support `-I` at all. `rgbasm -i` won't necessarily match the semantics of `gcc -I` all the time; `rgbasm -P/--preinclude` doesn't match `gcc -include` (since it doesn't count for `-MG`).

```suggestion
		// `-i` was the only short option for `--include` until `-I` was introduced
		// to better match the `-I dir` option of gcc and clang.
		// `-i` is now undocumented but still supported.
		case 'I':
```

Reply:
We have removed deprecated features from the docs while they were still supported before; the point of doing it like this is not to immediately spring up a warning, and give `-I` time to become adopted before we do so.

Reply:
Okay, so the long-term plan is to replace `-i` with `-I`; updated the comment accordingly.

Reply:
Yeah, looks good to me.